scripts/005-binutils-SPU.sh

```diff
@@ -42,7 +44,13 @@ cd ${BINUTILS}/build-spu
     --disable-werror \
     --with-gcc \
     --with-gnu-as \
-    --with-gnu-ld
+    --with-gnu-ld \
+    CFLAGS="-O3" \
+    CXXFLAGS="-g -O3" \
+    CCFLAGS_FOR_TARGET="-g -O3" \
+    CXXFLAGS_FOR_TARGET="-g -O3" \
+    GOCFLAGS_FOR_TARGET="-O3 -g" \
+    BOOT_CFLAGS="-g -O3" \
 
```

Comment:
Dont use -O3 here. SPU code needs to be as samll as possible. -Os and a few more other options are to go here for the TARGET flags. '-g' is already passed by default. And as a personal note: I prefer to have the flags before the '../configure' call.

Reply:
Why does the SPU code need to be as small as possible? So I believe it is best to leave it in the default

Reply:
Because on each SPU you only have 256Kb local store memory, which is used for executable code, program data and stack. So if the executable code gets big you don't have much left to operate on with.

Reply:
Ah yes, sorry, I forgot that detail